src/index.js

```diff
@@ -221,33 +195,42 @@ class IpfsRepo {
   exists (callback) {
     this.version.exists(callback)
   }
+}
 
-  /**
-   * Set the api address, by writing it to the `/api` file.
-   *
-   * @param {Multiaddr} addr
-   * @param {function(Error)} callback
-   * @returns {void}
-   */
-  setApiAddress (addr, callback) {
-    this._fsStore.put(apiFile, Buffer.from(addr.toString()), callback)
-  }
+module.exports = IpfsRepo
 
-  /**
-   * Returns the registered API address, according to the `/api` file in this respo.
-   *
-   * @param {function(Error, Mulitaddr)} callback
-   * @returns {void}
-   */
-  apiAddress (callback) {
-    this._fsStore.get(apiFile, (err, rawAddr) => {
-      if (err) {
-        return callback(err)
-      }
+function ignoringAlreadyOpened (cb) {
+  return (_err) => {
+    let err = _err
```

Comment:
A couple things here, its fairly standard to have a code property on an error in addition to message for this sort of situation (e.g. [fs.stats documentation](https://nodejs.org/dist/latest-v7.x/docs/api/fs.html#fs_fs_stat_path_callback)).

Also, the parameter should be called error for convention purposes, standard will do some enforcement around variables of that name specifically as well.

I would probably write this as:
```js
function ignoringAlreadyOpened (cb) {
  return (err) => err && err.code === 'AOPEN' ? callback() : callback(err)
}
```

This would imply you are adding the code `'AOPEN'` too of course. You don't want to really assume the error message is going to be English.